### PR TITLE
[exporter/loki] Remove deprecated configuration parameters

### PR DESCRIPTION
**Pull request by mar4uk**
**Description:** <Describe what has changed.>
Remove deprecated configuration parameters. The support of labels.{attributes/resource}, labels.record, tenant, and format configuration parameters are dropped in favor of attribute hints for configuring the exporter.

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15653
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15365

### CODE REVIEW

File: exporter/lokiexporter/exporter.go [hunk 1]
```diff
@@ -32,23 +34,27 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/loki"
 )
 
-type nextLokiExporter struct {
+const (
+	maxErrMsgLen = 1024
```
Comment (codeboten) [inline exporter/lokiexporter/exporter.go:38]: Is this used anywhere?

**Review by codeboten — COMMENTED** [review]
Pinging code owners: @gramidt @gouthamve @jpkrohling @kovrus

**Review by jpkrohling — COMMENTED** [review]
LGTM, this only needs clarification based on @codeboten's question.

File: exporter/lokiexporter/exporter.go (hunk 1 above)
Comment (mar4uk) [reply exporter/lokiexporter/exporter.go:38]: yes, it is used here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/36d526730a2453accb158dd7f4c73ce11f436993/exporter/lokiexporter/exporter.go#L112

Comment (codeboten) [reply exporter/lokiexporter/exporter.go:38]: Oh i see it was defined in a deleted file previously. I couldn't see it in the github diff :D Thanks!

**Review by codeboten — APPROVED** [review]

**Comment by mar4uk** [comment]
Closes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15653

**Pull request merged**